flutter/flutter
# Provide a way to change the default PopupMenuButton's icon size

## Description

Currently there is no way to change the PopupMenuButton's icon size. This PR fixes that.

## Related Issues

Fixes https://github.com/flutter/flutter/issues/67997

## Tests

I added the following tests:

- Verify that the size is properly applied
- Verify that the icon size defaults to the original value if not provided

## Checklist

Before you create this PR, confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.


## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].
